packages/flame/lib/src/components/isometric_tile_map_component.dart

```diff
@@ -105,11 +145,22 @@ class IsometricTileMapComponent extends PositionComponent {
 
   /// Same as getBlockRenderPosition but the arguments are exploded as integers.
```

Comment:
```suggestion
  /// Same as [getBlockRenderPosition] but the arguments are exploded as integers.
```

Know it is outside ur PR, but noticed that u did change it for the new method